pkg/volume/csi/expander.go

```diff
@@ -86,6 +86,15 @@ func (c *csiPlugin) nodeExpandWithClient(
 	if pv == nil {
 		return false, fmt.Errorf("Expander.NodeExpand failed to find associated PersistentVolume for plugin %s", c.GetPluginName())
 	}
+	nodeExpandSecrets := map[string]string{}
+	expandClient := c.host.GetKubeClient()
+	if csiSource.NodeExpandSecretRef != nil {
```

Comment:
I think we need to check the feature gate here too

Reply:
your means we should check the  CSINodeExpandSecret feature gate not  NodeExpandSecretRef?

Reply:
In addition to checking `NodeExpandSecretRef`, also check the feature gate.